src/main/java/dream/fcard/gui/controllers/displays/CreateDeckDisplay.java

```diff
@@ -12,14 +12,13 @@
 import javafx.fxml.FXMLLoader;
 import javafx.scene.control.Button;
 import javafx.scene.control.Label;
-import javafx.scene.control.ScrollPane;
 import javafx.scene.control.TextField;
-import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
 
 /**
  * This class is used for editing an existing deck as well as creating a new deck.
  */
-public class CreateDeckDisplay extends AnchorPane {
```

Comment:
What's the purpose of switching AnchorPanes to VBox?

Reply:
imo, it helps with alignment since you don’t have to manually designate the positions of elements (can define margins and paddings easily). if the vbox is wrapped in a scrollpane then the whole thing can just reflow and scroll, plus there isn’t really a solid need for anchoring objects in the anchor pane i think. i’ve tested this (visually) as i was making the changes, haven’t spotted any issues so far. lemme know if you have specific reasons for using anchorpane or any concerns!